cmake/options.cmake

```diff
@@ -65,6 +65,8 @@ option(BUILD_SHARED_LIBS "Whether to build shared libraries (like *.dll or *.so)
 
 option(ADD_HEADERS_AS_SOURCES "Whether to add headers as sources of a target or not. This is needed for some IDEs which wouldn't detect headers properly otherwise")
 
+option(OSQUERY_STATIC_BUILD "Whether to prefer linking static libraries or not")
```

Comment:
This option cannot be initialized here, because the file is included after the toolchain.cmake uses it.
This should be moved inside toolchain.cmake.